### PR TITLE
[IMP] Kanban: Add Inventory managament

### PR DESCRIPTION
**Pull request by etobella**
This PR allows to manage the kanban cards in an inventory process in order to check that no cards are lost or destroyed.

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
I got an error when accessing the Inventory menu after creating one:

```
Error:
Error: Unknown field state in domain
```

**Comment by AaronHForgeFlow** [comment]
Is it possible to create kanban inventories automatically?

**Comment by etobella** [comment]
Could you extend your question? I don't understand why you should need to create the inventory automatically

**Comment by AaronHForgeFlow** [comment]
Please dismiss my comment, I was not understanding the feature correctly.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Fuctional tests

File: stock_request_kanban/models/stock_inventory_kanban.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        for rec in self:
+            rec.missing_kanban_ids = rec.kanban_ids.filtered(
+                lambda r: r.id not in rec.scanned_kanban_ids.ids
+            )
+            rec.count_missing_kanbans = len(rec.missing_kanban_ids)
+
+    def _get_inventory_kanban_domain(self):
+        domain = []
+        if self.warehouse_ids:
+            domain.append(('warehouse_id', 'in', self.warehouse_ids.ids))
```
Comment (rousseldenis) [inline stock_request_kanban/models/stock_inventory_kanban.py]: Prefer use of expression.AND()

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Tested. Works as advertised. Thank you!

**Comment by JordiBForgeFlow** [comment]
@etobella This is good to go. Can you squash the commits?

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comment

**Comment by JordiBForgeFlow** [comment]
@etobella Can you squash your commits?

**Comment by etobella** [comment]
Squashed

**Pull request merged**